cmd/testing/base.go

```diff
@@ -29,18 +29,27 @@ import (
 
 type BaseSuite struct {
 	suite.Suite
-	Node     *node.Node
-	Client   *client.APIClient
-	ClientV2 clientv2.API
-	Host     string
-	Port     uint16
+	Node            *node.Node
+	Client          *client.APIClient
+	ClientV2        clientv2.API
+	Config          types.BacalhauConfig
+	Host            string
+	Port            uint16
+	AllowListedPath string
 }
 
 // before each test
 func (s *BaseSuite) SetupTest() {
 	logger.ConfigureTestLogging(s.T())
 
 	fsr, cfg := setup.SetupBacalhauRepoForTesting(s.T())
+	s.Config = cfg
+
+	// TODO: Update checker is configured with production default configs
+	//  and not respecting the test environment. This is a temporary fix
+	os.Setenv("BACALHAU_UPDATE_SKIPCHECKS", "true")
```

Comment:
We can use `config.KeyAsEnvVar` here instead of typing out the environment variable